### ISSUE TITLE
Search and filtering of results

### ISSUE THREAD

**Issue by rambotech** [NuGet/NuGetGallery#7953]
Nuget.org packages are dependent/support various .NET Framework and .NET Core versions. I don't see information on the search results which shows these.

Are there any plans to add advanced options for filtering such as:

- Selecting one or more Core/Framework versions to include (exclude the rest out)
- Selecting a date range when the package was uploaded or released.

The simplest feature would be a radio button for .NET Framework / .NET Standard / Both

Thanks.

**Comment by loic-sharma** [comment]
> Selecting one or more Core/Framework versions to include (exclude the rest out)
Yes that's definitely something we want to tackle. That is tracked by: https://github.com/NuGet/NuGetGallery/issues/3098

> Selecting a date range when the package was uploaded or released.
Interesting. Could you tell us more about your scenario and how filtering by upload/release date helps?

**Comment by rambotech** [comment]
The date scenario would be a way of looking for a package (or even an update) knowing approximately when it was released. This may be of no interest when using the NuGet Package Manager in VS or VS Code, but would be useful if vague information about a "package (say SSH) was released two weeks ago", with no further info.  This is a nice to have, compared to the main request: ensuring that a package found will support the framework desired.

**Comment by skofman1** [comment]
@rambotech , as @loic-sharma mentioned, filtering by framework is tracked by https://github.com/NuGet/NuGetGallery/issues/3098 and this is actually something we are starting to work on right now.

Regarding filtering by date, we have introduced Advanced Search that allow to order search result by upload date: https://devblogs.microsoft.com/nuget/advanced-search-on-nuget-org/
Is this helpful for the scenario you described?
